packages/ui-list/src/InlineList/props.ts

```diff
@@ -36,11 +36,22 @@ import type {
 } from '@instructure/shared-types'
 
 type InlineListOwnProps = {
-  children?: React.ReactNode // TODO: oneOf([InlineListItem])
+  /**
+   * Only accepts `<InlineList.Item>` as a child
+   */
+  children?: React.ReactNode
```

Comment:
That TODO from above is needed here, for typing `ChildrenPropTypes.oneOf([InlineListItem])` :)